seed-db.js

```diff
@@ -0,0 +1,190 @@
+const mongoose = require('mongoose');
+
+const reviewSchema = new mongoose.Schema({
+  reviewId: Number,
+  productId: Number,
+  overall: Number,
+  easeOfAssembly: Number,
+  valueForMoney: Number,
+  productQuality: Number,
+  appearance: Number,
+  worksAsExpected: Number,
+  recommended: Boolean,
+  title: String,
+  reviewText: String,
+  reviewerName: String,
+  reviewerId: Number,
+  date: Date
+});
+
+const Review = mongoose.model('Review', reviewSchema);
+
+var swap = (arr, i, j) => {
+  const temp = arr[i];
+  arr[i] = arr[j];
+  arr[j] = temp;
+};
+
+var shuffleDeck = function (deck) {
+  deck = deck.slice(); // don't want to mutate the input deck.
+  for (let i = 0; i < deck.length; i++) {
+    let j = i + Math.floor(Math.random() * (deck.length - i));
+    swap(deck, i, j);
+  }
```

Comment:
Good idea to shuffle reviewers, since each item won't have duplicate reviews from the same person.